ts/packages/ts/src/commands/UpsertCommand.ts

```diff
@@ -31,6 +31,7 @@ export class UpsertCommand extends Command<void> {
       .map<Config>(([key, value]) => {
         const cfgu = schemaContents[key];
 
+        let AllAliases: string[] = [];
```

Comment:
Why would create this here? for every config?
If at-all this should be declared before the loop